src/main/java/meteordevelopment/meteorclient/systems/modules/movement/Scaffold.java

```diff
@@ -159,88 +170,72 @@ public class Scaffold extends Module {
     );
 
     private final BlockPos.Mutable bp = new BlockPos.Mutable();
-    private final BlockPos.Mutable prevBp = new BlockPos.Mutable();
-
-    private boolean lastWasSneaking;
-    private double lastSneakingY;
 
     public Scaffold() {
         super(Categories.Movement, "scaffold", "Automatically places blocks under you.");
     }
 
-    @Override
-    public void onActivate() {
-        lastWasSneaking = mc.options.sneakKey.isPressed();
-        if (lastWasSneaking) lastSneakingY = mc.player.getY();
-    }
 
     @EventHandler
     private void onTick(TickEvent.Pre event) {
         if (onlyOnClick.get() && !mc.options.useKey.isPressed()) return;
 
+        Vec3d vec = mc.player.getPos().add(mc.player.getVelocity()).add(0, -0.75, 0);
```

Comment:
is 0.75 for gravity or something?

Reply:
This value determines how far below the player the blocks are placed, so logically it should be the smallest value greater than -1 (-0.999...).
However, setting it to -0.75 avoids placing blocks under the bridge like this:

https://github.com/MeteorDevelopment/meteor-client/assets/63475640/eb6e9f40-4592-40cb-ba4d-cf62742f1489